### PR TITLE
Update Unqiue wording differences post launch and other changes.

### PR DESCRIPTION
**Pull request by QuickStick123**
Fixes https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/4971 https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/5006

This includes the suggest/started changes completed in https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/2536

> 3.18.0: The following Unique Items will now be found with both Shaper- and Elder-Influence: Disintegrator, Indigon, Voidfletcher, Voidforge, The Eternity Shroud, and Replica Eternity Shroud. This change does not apply to existing versions of these Unique Items. 

I updated all the items but have not added legacy variants for these as would need to add support.

There is a bug which is much more noticable with multibase/level requirement uniques where they don't update.

["^(%d+)"] = "BASE", was added to get # life on block working as it no longer has any + sign.
It is a very broad form which may cause issues.

### CODE REVIEW

File: src/Data/Uniques/claw.lua [hunk 1]
```diff
@@ -112,54 +113,44 @@ Variant: Pre 3.19.0
...
+{variant:4,5}2% of Physical Attack Damage Leeched as Life
+{variant:1}+10% Chance to Block Attack Damage while Dual Wielding Claws
+{variant:2,3,4,5}+8% Chance to Block Attack Damage while Dual Wielding Claws
 {variant:1}(80-120)% increased Physical Damage
-{variant:2,3}(140-180)% increased Physical Damage
+{variant:2,3,4,5}(140-180)% increased Physical Damage
 {variant:1}Adds 1 to 50 Lightning Damage
 {variant:2,3}Adds 1 to 80 Lightning Damage
+{variant:4,5}Adds 1 to 200 Lightning Damage
+{variant:6}Adds 1 to (500-600) Lightning Damage
```
Comment (ifnjeff) [inline src/Data/Uniques/claw.lua:147]: Patch notes have been updated, new version adds 1 to 600-700 lightning

Comment (QuickStick123) [reply src/Data/Uniques/claw.lua]: Updated patch notes for essentia and divinaurs

File: src/Data/Uniques/body.lua [hunk 2]
```diff
@@ -855,15 +855,16 @@ Implicits: 0
 {variant:1,2}+10% to all Elemental Resistances
 {variant:3,4,5}+15% to all Elemental Resistances
 {variant:1,2,3,4}Gain an Endurance Charge when you take a Critical Strike
-{variant:5}Gain up to Maximum Endurance Charge when you take a Critical Strike
+{variant:5}Gain up to maximum Endurance Charges when you take a Critical Strike
 {variant:1,2,3}Regenerate 2% of Life per Second while on Low Life
-Share Endurance Charges with nearby party members
-{variant:5}Your nearby party members' maximum Endurance Charges are equal to yours.
+{variant:1,2,3,4}Share Endurance Charges with nearby party members
+{variant:5}Your nearby party members maximum Endurance Charges is equal to yours
```
Comment (Wires77) [inline src/Data/Uniques/body.lua:861]: Grammar like this bothers me, but it matches in-game, so c'est la vie

**Review by Wires77 — APPROVED** [review]
I've let perfect be the enemy of done on this one for too long, and what this encompasses looks good.  Maybe someday I'll be able to merge in my automatic script, heh.

**Comment by QuickStick123** [comment]
I have reviewed this my self at least 3 times so I am fairly confident it is fine.

**Comment by Wires77** [comment]
> I have reviewed this myself at least 3 times so I am fairly confident it is fine.

Same, honestly.  There wasn't much new since I reviewed it 3 months ago

**Pull request merged**